Makefile

```diff
@@ -146,14 +146,32 @@ fmt:
 
 MANIFESTS=manifests/$(LOGGING_VERSION)
 
+# Generate bundle manifests and metadata, then validate generated files.
```

Comment:
So I just moved the modifications to the script `./hack/generate-crd.sh` instead of the Makefile.
reason is when it was in the Makefile, and `USE_IMAGE_DIGESTS` is set to true, `make bundle` fails with the following error: 

> FATA[0000] Error generating bundle manifests: Operator bundle may contain only 1 CSV file, but contains more